src/api/app/views/webui2/webui/request/_show_overview.html.haml

```diff
@@ -0,0 +1,13 @@
+%h3
+  %p
+    = pagetitle
+    = "(#{state})"
+#description-text
+  - if description.blank?
+    %i No description set
+  - else
+    = description_wrapper(toggle_sliced_text(description))
+- if can_add_reviews
+  %p
+    %i.fas.fa-plus-circle.text-primary
```

Comment:
This should instead be:

```
= link_to('Add a review', add_reviewer_dialog_path(number: number), remote: true) do
  %i.fas.fa-plus-circle.text-primary
  Add a Review
```

So the link is properly capitalized and the font weight is normal (not bold as it is right now)

Reply:
didnt work, maybe passing the block to the method link_to, you have to change something?

Reply:
Yep. In that case you can drop the first parameter, since that's essentially the block.

```ruby
= link_to(add_reviewer_dialog_path(number: number), remote: true) do
  %i.fas.fa-plus-circle.text-primary
  Add a Review
```

Reply:
And also the `remote: true` is not needed anymore

Reply:
Oh right, I forgot to remove the string `'Add a review'`. @bgeuken's version is the right one.

Reply:
i saw the comments now.. done. Could you please check again @dmarcoux ? Thanks anyway @bgeuken  and @DavidKang